lib/cancan/ability.rb

```diff
@@ -252,6 +252,20 @@ def merge(ability)
       self
     end
 
+    def list_can(filter_action = nil)
```

Comment:
I'm not sure we need list_can, or a filter_action. As well as the same for list_cannot. It just wraps a hash we have to build in memory anyway.